h/static/styles/partials-v2/_annotation-card.scss

```diff
@@ -68,12 +68,22 @@
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
+  align-items: baseline;
 }
 
-.annotation-card__incontext-link {
-  height: 10px;
-  width: 10px;
-  color: $grey-5;
+.annotation-card__footer-link {
+  margin-left: 15px;
+  color: $grey-4;
+  height: 16px;
+  width: 16px;
+}
+
+.annotation-card__footer .annotation-card__incontext-link {
```

Comment:
I don't think we need this descendant selector, given that the incontext link is only going be shown in the footer. Even if we did, the convention would be to use a modifier class in most cases (ie. `some-thing--variation` rather than `some-parent some-thing`)